open_api_framework/conf/base.py

```diff
@@ -0,0 +1,662 @@
+import os
+import warnings
+from typing import Callable
+
+from django.urls import reverse_lazy
+
+import sentry_sdk
+from corsheaders.defaults import default_headers as default_cors_headers
+from log_outgoing_requests.formatters import HttpFormatter
+from notifications_api_common.settings import *  # noqa
+
+from .utils import (
+    config,
+    get_django_project_dir,
+    get_project_dirname,
+    get_sentry_integrations,
+    strip_protocol_from_origin,
+)
+
+PROJECT_DIRNAME = get_project_dirname()
+
+# Build paths inside the project, so further paths can be defined relative to
+# the code root.
+DJANGO_PROJECT_DIR = get_django_project_dir()
+BASE_DIR = os.path.abspath(
```

Comment:
Should we move from `os.path` to `Pathlib`?

Reply:
Seems like a good idea :+1: